gigantum/gigantum-client
# 1669 block visibility change

## Pull Request for Issue #1669 #1670

**Describe changes introduced by this PR (Bugs fixed, features added, docs updated, etc.)**:
Adds blocking on visibility change, and handles failures when visibility fails due to server being in backup mode.
Adds backup icon and info icon to server name on side panel.

**Please check if the PR fulfills these requirements**

  - PR is set to merge to integration
  - PR title is descriptive, as this becomes the commit message on squash and merge.
  - Any related issues are added to the extended commit message during squash and merge.
  - Tests have been added (for bug fixes/features)
  - Docs have been added/updated (for bug fixes/features)

**QA Test Steps and Exit Criteria**:
1. Load page not in backup mode.
2. switch server to backup mode
3. change visibility and see that backup failure is captured.
4. remote operations should now be locked out. Server backup icon should be spinning in side panel with info icon.
5. reload and make sure everything is still locked out.
6. turn off backup mode and wait for ui to update and unlock remote operations.
